infra.bs

```diff
@@ -547,6 +548,38 @@ ordered sets; implementations can optimize based on the fact that the order is n
 <a for=list>prepend</a> operation otherwise.
 
 
+<h3 id=tuples>Tuples</h3>
+
+<p>A <dfn export>tuple</dfn> is a specification type consisting of a finite ordered
```

Comment:
No, @domenic borrowed it from ECMAScript I think. We might want to formalize that too.